.github/workflows/tests.yml

```diff
@@ -36,6 +36,7 @@ jobs:
             - name: Check formatting
               run: pnpm format:check
     tests:
+        if: false # This test job is currently disabled due to failing redirects tests.
```

Comment:
If possible, lets try to disable specific tests. If not, well, not ideal, we might prfioritize fixing this soon. Good enough to disable for now

Reply:
as discussed internally and mentioned by Torsten, format and build have their own tests. Redirect is a standalone test via Playwright.